CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
   - `array_max`
   - `explode_outer`
 - Added support for pure Python packages specified via `Session.add_requirements` or `Session.add_packages`, now usable in stored procedures and UDFs even if packages are not present on the Snowflake Anaconda channel.
+- Added Session parameter `custom_packages_upload_enabled` to enable the feature mentioned above (defaults to `False`).
```

Comment:
```suggestion
- Added support for pure Python packages specified via `Session.add_requirements` or `Session.add_packages`, now usable in stored procedures and UDFs even if packages are not present on the Snowflake Anaconda channel.
  - Added Session parameter `custom_packages_upload_enabled` to enable the feature mentioned above (defaults to `False`).
```
How about moving this in a sub bullet point?

Reply:
Thanks, changed this. Not sure where to document info about the parameters; where would I add that?